### PR TITLE
add travis image to README

### PR DESCRIPTION
**Pull request by marutosi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[<img src="https://travis-ci.org/rabbit-shocker/rabbit.svg?branch=master" alt="Build Status" />](https://travis-ci.org/rabbit-shocker/rabbit)
```
Comment (cosmo0920) [inline README.md]: How about using following style?

```log
[![Build Status](https://travis-ci.org/rabbit-shocker/rabbit.svg?branch=master)](https://travis-ci.org/rabbit-shocker/rabbit)
```

**Comment by marutosi** [comment]
I rebased.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[![Build Status](https://travis-ci.org/rabbit-shocker/rabbit.svg?branch=master)](https://travis-ci.org/rabbit-shocker/rabbit)
+
+See doc [doc/](doc/) or http://rabbit-shocker.org/.
```
Comment (kou) [inline README.md]: It seems that "doc" is garbage.

**Comment by marutosi** [comment]
> It seems that "doc" is garbage.

No. It is link to doc/.
Before:
https://github.com/marutosi/rabbit/blob/6d98ac83e5c830bc2317783bbd3206c988390538/README.md
After:
https://github.com/marutosi/rabbit/blob/1485052e8558ccf5294eaecec95d52a170277e35/README.md

**Comment by cosmo0920** [comment]
It seems that `doc [doc/](doc/)` is a bit of redundant.

I think following patch is more sophisticated, though.

```diff
-See doc [doc/](doc/) or http://rabbit-shocker.org/.
+See [doc/](doc/) or http://rabbit-shocker.org/.
```

**Comment by marutosi** [comment]
I rebased.

**Comment by cosmo0920** [comment]
Thanks.
It looks good to me.

But, I do not prefer force push.
Because it makes hard to follow the steps along commits.

**Comment by kou** [comment]
I like "force push" for *pull request*. If we can't understand something from merged commits, the pull request or each commit is large. We should split the pull request to smaller pull requests instead of avoiding "force push".

There is no problem what @marutosi did. Thanks for your contribution!
